isort/setuptools_commands.py

```diff
@@ -16,7 +16,8 @@ class ISortCommand(setuptools.Command):
     """
 
     description = "Run isort on modules registered in setuptools"
-    user_options: List[Any] = []
+    # Potentially unused variable - check if can be safely removed
+    user_options: list[Any] = [] # type: ignore
```

Comment:
```suggestion
    user_options: list[Any] = [] # type: ignore[error-code]
```

Nit, but I always add the error code that I'm ignoring so that I don't accidentally ignore a new error code after a `mypy` bump. Could you add the specific code?

Reply:
Added.